lib/sles4sap_publiccloud.pm

```diff
@@ -530,4 +531,40 @@ sub create_instance_data {
     return \@instances;
 }
 
+=head2 deployment_name
+
+    Return a string to be used as value for the deployment_name variable
+    in the qe-sap-deployment.
+
+=cut
+
+sub deployment_name {
+    return qesap_calculate_deployment_name(get_var('PUBLIC_CLOUD_RESOURCE_GROUP', 'qesaposd'));
+}
+
+=head2 delete_network_peering
+
+    Delete network peering between SUT created with qe-sa-deployment
+    and th eIBS Mirror. Function is generic over all the Cloud Providers
+
+=cut
+
+sub delete_network_peering {
+    record_info('Peering cleanup', 'Executing peering cleanup (if peering is present)');
+    if (is_azure) {
+        my $rg = qesap_az_get_resource_group();
+        my $ibsm_rg = get_required_var('IBSM_RG');
+        # Check that required vars are available befor delleting the peering
+        if (defined $rg && $rg ne '' && defined $ibsm_rg && $ibsm_rg ne '') {
```

Comment:
`if ($rg && $ibsm_rg)`

Wouldn't it work like this?

Reply:
I just inherited and moved here the same from https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17329/files#diff-5404750314cf685b060d425b85a04de2cdfb000225014d52ba83700abe58b2bdL38

Yes, check with `defined` is not needed as variables are defined few lines before.